paddle/fluid/operators/slice_op.cc

```diff
@@ -34,18 +34,31 @@ class SliceOp : public framework::OperatorWithKernel {
     PADDLE_ENFORCE_EQ(ctx->HasOutput("Out"), true,
                       "Output (Out) of slice op should not be null.");
     auto x_var_type = ctx->GetInputsVarType("Input")[0];
-    if (x_var_type == framework::proto::VarType::LOD_TENSOR_ARRAY &&
-        ctx->IsRuntime()) {
-      // If the var type of input is LOD_TENSOR_ARRAY,
-      // the output shape is determined by SliceKernel:Compute in runtime.
-      return;
+    auto axes = ctx->Attrs().Get<std::vector<int>>("axes");
+    if (x_var_type == framework::proto::VarType::LOD_TENSOR_ARRAY) {
+      PADDLE_ENFORCE_EQ(axes.size(), 1,
+                        platform::errors::InvalidArgument(
+                            "The size of axes must be 1 when the Input of "
+                            "SliceOp is LoDTensorArray, "
+                            "but received %d.",
+                            axes.size()));
+      if (ctx->IsRuntime()) {
+        // If the var type of input is LOD_TENSOR_ARRAY,
+        // the output shape is determined by SliceKernel:Compute in runtime.
+        return;
+      } else {
+        // NOTE: A better way is needed to get accurate dims of tensor array.
```

Comment:
Not important. Better add your name in NOTE, so other RDs can discuss with you if they are confusing about it.

Reply:
> Not important. Better add your name in NOTE, so other RDs can discuss with you if they are confusing about it.

Thanks. I will add my name next PR.